### DISCUSSION TITLE
Use BW button for an RDS reset in AF/EON window

### DISCUSSION THREAD

**Discussion by andimik** [PE5PVB/TEF6686_ESP32#92] [Ideas]
Due to the fact, that in AF/EON window we cannot change the frequency, the **BW button** is "dead" now and has (currently) no function. (Hint: **MODE button** will switch to `2/2` resp. back to `1/2` for huge AF >33 frequencies and **BAND button** will close the window and will go back to advanced RDS window)

So I suggest to use **BW button** for a RDS reset in case of errors.

**Comment by PE5PVB** [comment]
There is already a check for 0 errors in the last repository, so it should be error free. I have other plans for that button.
